ahfe_tutorial/python_tutorial.ipynb

```diff
@@ -0,0 +1,930 @@
+{
```

Comment:
<p>I've been trying to think about the information flow here, how about something like this:</p><p> </p><p>"</p><p>Finally now that we've run our simulations, let's go ahead and gather the free&nbsp;energies for both phases.</p><p> </p><p><strong>Python API</strong></p><p> </p><p>If you executed the simulations using the Python API, you will have generated a <code>dag_results</code> object. You can analyze these results by calling the Protocols' <code>gather()</code> method. This takes a <strong>list</strong> of completed DAG results and returns a <code>AbsoluteSolvationProtocolResult</code> which can return a free energy estimate and uncertainty by calling the <code>get_estimate()</code> and <code>get_uncertainty()</code> methods."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>